src/index.js

```diff
@@ -31,3 +31,14 @@ registerBlockType( metadata.name, {
 	 */
 	edit: Edit,
 } );
+
+registerBlockVariation("twentybellows/query-filter",{
```

Comment:
Given that the query filter block is added in code and not via the block menu, I'm not sure this actually does anything useful.